src/views/Dashboard.tsx

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
 import DashboardHeader from '../components/DashboardHeader';
 import SideBar from '../components/sidebar/SideBar';
+import DashboardList from '../components/dashboard/dashboard-list/DashboardList';
+
+import './Dashboard.scss';
+
 
 const Dashboard: React.FunctionComponent = () => {
     return (
         <div className="section view">
+            <h1>Dashboard</h1>
+            <DashboardHeader />
+            <DashboardList />
```

Comment:
DashboardHeader and DashboardList should go inside of the second column div (not the one with SideBar)

You will have to remove the existing h1 and DashboardHeader from there